hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/compact/TestHoodieCompactor.java

```diff
@@ -195,19 +195,18 @@ public void testWriteStatusContentsAfterCompaction() throws Exception {
       String newCommitTime = "100";
       writeClient.startCommitWithTime(newCommitTime);
 
-      List<HoodieRecord> records = dataGen.generateInserts(newCommitTime, 100);
+      List<HoodieRecord> records = dataGen.generateInserts(newCommitTime, 1000);
       JavaRDD<HoodieRecord> recordsRDD = jsc.parallelize(records, 1);
       writeClient.insert(recordsRDD, newCommitTime).collect();
 
       // Update all the 100 records
```

Comment:
100 -> 1000?

Reply:
The idea is to ensure that the scan times are in milliseconds as opposed to microsecs, so increasing the load.